### DISCUSSION TITLE
SSD Boot Drive Exclusion in Pool Creation

### DISCUSSION THREAD

**Discussion by rbachar** [trapexit/mergerfs#1325] [Q&A] [answered]
Hey everyone,

I've set up my Raspberry Pi 5 to boot from a 2TB PCIe NVMe drive and have OMV7 installed on it. Additionally, I've connected a 1TB USB3 SSD to the Raspberry Pi. However, when I try to create a new pool with these two drives and save the configuration, only my USB SSD drive appears, as illustrated in the attached photos. Is there a step I'm missing? Does mergerfs have certain restrictions, or could this be a bug?
Thanks!
![2024-04-08_09-58-55](https://github.com/trapexit/mergerfs/assets/80811321/810607d9-ddff-462e-b60a-214bffc9a9b0)
![2024-04-08_09-59-06](https://github.com/trapexit/mergerfs/assets/80811321/643925f2-d7fd-4ca0-abdd-599c6aaf1c13)

**Comment by trapexit** [comment]
Sorry, but you are using OMV plugin which I don't own or am all that familiar with. That's just some 3rd party config interface. Not mergerfs itself. You'll need to ask the OMV folks about that. It's certainly not an issue with mergerfs.

**Reply by rbachar** [reply]
Thank you

**Comment by ryecoaaron** [comment]
@rbachar Please post on the [OMV forum](https://forum.openmediavault.org/index.php?unread-thread-list/) or the [plugin's issues](https://github.com/OpenMediaVault-Plugin-Developers/openmediavault-mergerfs/issues)

It looks like you are trying to put the root filesystem (/) in the pool.  The plugin prevents this.  You will have use a subfolder.

**Reply by rbachar** [reply, marked as answer]
Thank you!
